packages/phisherman/src/lib/Phisherman.ts

```diff
@@ -0,0 +1,82 @@
+import { fetch, FetchMethods, FetchResultTypes, QueryError } from '@sapphire/fetch';
+import type { PhishermanReportType, PhishermanReturnType } from './PhishermanTypes';
+
+let storedApiKey: string;
+
+/**
+ * Checks if a link is detected as a scam or phishing link by phisherman.
+ * @param domain The domain to check.
```

Comment:
Document optional apiKey param

Reply:
```
@param apiKey optionally pass a Phiserman API key for making this request. This will default to {@link storedApiKey}, which can be configured through {@link setApiKey}.
```